demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
Alrighty! Thanks for taking the time to explain the reasoning to me, much appreciated. I've reverted the relevant bits.